noxfile.py

```diff
@@ -249,3 +263,16 @@ def format(session):
     session.run("black", *LINT_FILES)
     session.run("blackdoc", *LINT_FILES)
     session.run("isort", *LINT_FILES)
+
+
+@nox.session
+def clean(session):
```

Comment:
I don't know if you want but you can may also want to delete the folder "rendered folder" "docs/src/examples".

Reply:
I did not thought about that you can add that, sure

Reply:
I actually was thinking of removing _specifically_ that folder, and not stray files in `examples/` that could be for instance test jupyter notebooks or similar. it is the stray files in docs/src/examples/ that generate the most problems in my experience